cmp-shared/cmp_shared.podspec

```
@@ -51,5 +51,4 @@ Pod::Spec.new do |spec|
             SCRIPT
         }
     ]
-    spec.resources = ['build/compose/cocoapods/compose-resources']
-end
\ No newline at end of file
+    spec.resources = ['build/compose/cocoapods/compose-resources']
```

RuboCop (1.88.0)

[fatal] unexpected token $end
(Using Ruby 3.3 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)